components/button/button.tsx

```diff
@@ -269,6 +270,7 @@ const InternalButton: React.ForwardRefRenderFunction<unknown, ButtonProps> = (pr
       </a>
     );
   }
+  const dataOrAriaProps = getDataOrAriaProps(props);
```

Comment:
This code is useless. `rest` already pass the props: https://codesandbox.io/s/anniuleixing-antd4141-forked-wj54d?file=/index.js